atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableRemappingKeyValueService.java

```diff
@@ -156,6 +161,10 @@ private Set<TableReference> getShortTableReferencesForExistingTables(Set<TableRe
                 tableNames.add(tableMapper.getMappedTableName(tableRef));
             } catch (TableMappingNotFoundException e) {
                 // Table does not exist - do nothing
+                log.debug(
+                        "Could not find short table reference for an existing table",
+                        UnsafeArg.of("tableRef", tableRef),
```

Comment:
technically you can `LoggingArgs.tableRef(...)` this probably to get the name if it's safe? Though I'm guessing in the contexts you need this it's not going to be safe

Reply:
Cool, swapped for LoggingArgs